.github/workflows/test-docs.yml

```diff
@@ -3,40 +3,13 @@ on:
   pull_request:
     paths:
       - "docs/**"
+  workflow_dispatch:
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '16.15.1'
-
-      - run: rm -rf build
-
-      - uses: pnpm/action-setup@v2
-        name: Install pnpm
-        id: pnpm-install
-        with:
-          version: 7.5.0
-          run_install: false
-
-      - name: Get pnpm store directory
-        id: pnpm-cache
-        shell: bash
-        run: |
-          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
-
-      - uses: actions/cache@v3
-        name: Setup pnpm cache
-        with:
-          path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
-          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-store-
-
-      - name: Install dependencies
-        run: pnpm install
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-project
```

Comment:
Aren't the steps of the job `test` the same as the workflow defined `workflow_dispatch`? Should reuse the workflow for the `test` job?
Not sure if I miss anything but to me this looks like it is being executing the whole steps of the `workflow_dispatch` twice at the end.
This would happen in the other ocurrences where these same changes were done.

Reply:
I'm not sure i totally follow you question :( Could elaborate a bit? This addition of workflow dispatch was added so we can call this workflow manually from the UI and not only on PR's.

the job test will run only ones either on a PR or, manually triggered from UI. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Reply:
My mistake when reviewing. I read it as `workflow_dispatch` was a workflow defined by us that executed some steps, and not as a property of the workflow as it is. That made me think that we were double executing steps. 😓